README.md

```diff
@@ -0,0 +1,10 @@
+# Insomnia for Flatpak
```

Comment:
No need to have it here unless you feel strongly about it. IMO it just duplicates existing documentation.

Reply:
Nah, this was just here because this repository as experimental for a long time.